apps/web/src/pages/SearchRequests/ViewSearchRequestPage/components/UpdateSearchRequest.tsx

```diff
@@ -78,112 +86,170 @@ export const UpdateSearchRequestForm = ({
+                <Submit
                   color="primary"
                   disabled={isSaving}
-                  onClick={() => {
-                    handleSaveNotes(getValues());
-                  }}
-                >
-                  {isSaving
-                    ? intl.formatMessage(commonMessages.saving)
-                    : intl.formatMessage({
-                        defaultMessage: "Save Notes",
-                        id: "DRsBYY",
-                        description:
-                          "Button label displayed on the search request form which saves the users personal notes.",
-                      })}
-                </Button>
+                  isSubmitting={isSaving}
+                  text={intl.formatMessage({
+                    defaultMessage: "Save status change",
+                    id: "B6SqfX",
+                    description:
+                      "Button label displayed that saves the users status selection.",
+                  })}
+                  submittedText={intl.formatMessage({
+                    defaultMessage: "Save status change",
+                    id: "B6SqfX",
+                    description:
+                      "Button label displayed that saves the users status selection.",
+                  })}
+                />
               </div>
             </div>
-            <div data-h2-margin="base(0, 0, x1, 0)">
-              <Button
-                color="secondary"
-                mode="inline"
-                onClick={() => {
-                  navigate(paths.searchRequestTable());
-                }}
-                data-h2-margin="base(x1, x1, 0, 0)"
-              >
-                {intl.formatMessage({
-                  defaultMessage: "Back to requests",
-                  id: "O8nHiQ",
-                  description:
-                    "Button label displayed on the search request form which returns the user back to requests.",
-                })}
-              </Button>
-              <Submit
-                text={intl.formatMessage({
-                  defaultMessage: "Mark this request as done",
-                  id: "UNScjB",
-                  description:
-                    "Button label displayed on the search request form which changes request status to done.",
-                })}
-              />
-            </div>
           </form>
         </FormProvider>
       </div>
-    </section>
+      <div data-h2-margin="base(0, 0, x1, 0)">
+        <Button
+          color="secondary"
+          mode="inline"
+          onClick={() => {
+            navigate(paths.searchRequestTable());
+          }}
+          data-h2-margin="base(x1, x1, 0, 0)"
+        >
+          {intl.formatMessage({
+            defaultMessage: "Back to requests",
+            id: "O8nHiQ",
+            description:
+              "Button label displayed on the search request form which returns the user back to requests.",
+          })}
+        </Button>
```

Comment:
I think this would be better as a link since all it does is navigate.

```tsx
<Link href={paths.searchRequestTable()} mode="inline" color="secondary">...</Link>
```

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7210/commits/9ce5563e4df5b2b960fbf2533b0b8158f69b47f5